packages/tools/client-debugger/client-debugger/src/FluidClientDebugger.ts

```diff
@@ -39,6 +42,7 @@ import { FluidClientDebuggerProps } from "./Registry";
  * - {@link ConnectContainerMessage}: When received (if the container ID matches), the debugger will connect to the container.
  * - {@link DisconnectContainerMessage}: When received (if the container ID matches), the debugger will disconnect from the container.
  * - {@link CloseContainerMessage}: When received (if the container ID matches), the debugger will close the container.
+ * - {@link AudienceEventMessage}: When received (if the container ID matches), the debugger will send the Audience data consisting of contaierId, current connected members, and history of audience members
```

Comment:
This should be `GetAudienceMessage`, right? We should also document the `AudienceEventMessage` in the outgoing messaging section here, and we can update these docs to simply note that the `AudienceEventMessage` is posted in response to receiving `GetAudienceMessage` (to reduce docs duplication).

Reply:
Changed to `GetAudienceMessage` including the information of broadcasting `AudienceSummaryMessage` (before `AudienceEventMessage`)